graph/R-GAT/tools/accuracy_igbh.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import numpy as np
+import torch
+import json
+import os
+
+
+def get_args():
+    """Parse commandline."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--mlperf-accuracy-file", required=True, help="path to mlperf_log_accuracy.json"
+    )
+    parser.add_argument(
+        "--dataset-path",
+        default="igbh",
+        help="Path to IHGB dataset",
+    )
+    parser.add_argument(
+        "--dataset-size",
+        default="full",
+        choices=["tiny", "small", "medium", "large", "full"]
+    )
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        help="verbose messages")
+    parser.add_argument(
+        "--output-file", default="coco-results.json", help="path to output file"
+    )
+    parser.add_argument(
+        "--dtype",
+        default="int32",
+        choices=["float32", "int32", "int64"],
+        help="data type of the label",
+    )
+    args = parser.parse_args()
+    return args
+
+def load_labels(base_path, dataset_size, use_label_2K = True):
+    # load labels
+    paper_nodes_num = {
+        "tiny": 100000,
+        "small": 1000000,
+        "medium": 10000000,
+        "large": 100000000,
+        "full": 269346174,
+    }
+    label_file = (
+        "node_label_19.npy" if not use_label_2K else "node_label_2K.npy"
+    )
+    paper_lbl_path = os.path.join(base_path, "paper", label_file)
+
+    if dataset_size in ["large", "full"]:
+        paper_node_labels = torch.from_numpy(
+            np.memmap(
+                paper_lbl_path, dtype="float32", mode="r", shape=(paper_nodes_num[dataset_size])
+            )
+        ).to(torch.long)
+    else:
+        paper_node_labels = torch.from_numpy(
+            np.load(paper_lbl_path)).to(
+            torch.long)
+    labels = paper_node_labels
+    val_idx = torch.load(
+                os.path.join(
+                    base_path,
+                "val_idx.pt"))
```

Comment:
`val_idx.pt` is present inside `base_path/dataset-size/processed/`,  I think including [this](https://github.com/mlcommons/inference/blob/e9bed37c7b7275755924ae368b60832ade863da4/graph/R-GAT/igbh.py#L177)  line should fix.

Reply:
updated it to: https://github.com/mlcommons/inference/blob/7ba22388b2e5247740383be3c4f83ec588d4bb92/graph/R-GAT/tools/accuracy_igbh.py#L67-L70